tests/libwait/wait.go

```diff
@@ -216,3 +216,14 @@ func WaitForMigrationToDisappearWithTimeout(migration *v1.VirtualMachineInstance
 		return err
 	}, seconds, 1*time.Second).Should(gomega.MatchError(errors.IsNotFound, "k8serrors.IsNotFound"), fmt.Sprintf("migration %s was expected to disappear after %d seconds, but it did not", migration.Name, seconds))
 }
+
+func WaitUntilVMReadyWithTimeout(vm *v1.VirtualMachine, seconds int) {
+	virtClient, err := kubecli.GetKubevirtClient()
+	gomega.ExpectWithOffset(1, err).ToNot(gomega.HaveOccurred())
+
+	gomega.EventuallyWithOffset(1, func() bool {
```

Comment:
There's a BeReady matcher that exists today in `tests/framework/matcher/status.go`

Reply:
done